src/app/topbar/topbar.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-topbar',
+  templateUrl: './topbar.component.html',
+  styleUrls: ['./topbar.component.css']
+})
+export class TopbarComponent implements OnInit {
```

Comment:
One way to decide whether to have a component or not is to see if it has some individual behaviour, I don't find any individual behaviour here.